python-frontend/src/main/java/org/sonar/python/semantic/v2/TypeInferenceV2.java

```diff
@@ -130,11 +131,10 @@ private static void inferTypesAndMemberAccessSymbols(Tree scopeTree,
   }
 
   private static void flowSensitiveTypeInference(ControlFlowGraph cfg, Set<SymbolV2> trackedVars, PropagationVisitor propagationVisitor) {
-    // TODO: infer parameter type based on type hint or default value assignement
+    // TODO: infer parameter type based on default value assignement
     var parameterTypes = trackedVars
       .stream()
-      .filter(s -> s.usages().stream().map(UsageV2::kind).anyMatch(UsageV2.Kind.PARAMETER::equals))
-      .collect(Collectors.toMap(SymbolV2::name, v -> PythonType.UNKNOWN));
+      .collect(Collectors.toMap(SymbolV2::name, TypeInferenceV2::getParameterType));
```

Comment:
Does that mean that non-parameters will now be set to `PythonType.UNKNOWN`?
Could this lead to any possible problem?